tests/unit/test__helpers.py

```diff
@@ -1036,3 +1036,665 @@ def test_default_isolation_and_merge_options_isolation_unspecified(self):
         )
         result = self._callFUT(default, merge)
         self.assertEqual(result, expected)
+
+
+class Test_interval(unittest.TestCase):
+    def _callFUT(self, *args, **kw):
+        from google.cloud.spanner_v1._helpers import _make_value_pb
+
+        return _make_value_pb(*args, **kw)
+
+    def test_basic_interval(self):
+        from google.protobuf.struct_pb2 import Value
```

Comment:
nit: Move the imports to class level as all the tests uses the same imports.

Reply:
done